### PR TITLE
11.0 mig website_no_crawler

### PR DESCRIPTION
**Pull request by ernestotejeda**
cc @Tecnativa

### CODE REVIEW

File: website_no_crawler/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (simahawk) [inline website_no_crawler/README.rst]: you could update READM according to new specs https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: website_no_crawler/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2016 B-informed B.V.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Alter robots.txt disallow indexing',
+    'summary': 'Disables robots.txt for indexing by webcrawlers like Google',
+    'license': 'AGPL-3',
+    'version': '11.0.1.0.0',
+    'author': "B-Informed B.V.,Odoo Community Association (OCA)",
+    'category': 'Website',
```
Comment (simahawk) [inline website_no_crawler/__manifest__.py]: website: https://github.com/OCA/website

File: website_no_crawler/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    'depends': [
+        'website',
+    ],
+    'data': [
+        'views/disable_robots.xml',
+    ],
+    'demo': [],
```
Comment (simahawk) [inline website_no_crawler/__manifest__.py]: drop empty keys

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by rafaelbn** [comment]
cc @Tecnativa

**Review by pedrobaeza — APPROVED** [review]
I have also put the new README fragments + attend the comments.

**Review by simahawk — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

**Review by yajo — APPROVED** [review]

**Pull request merged**